front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -178,8 +182,29 @@ const ManchetteWithSpaceTimeChartWrapper = ({
     }));
   });
 
+  const waypointMenuData = useWaypointMenu(
+    waypointsPanelData?.filteredWaypoints,
+    waypointsPanelData?.setFilteredWaypoints
+  );
```

Comment:
Nit: you can simply give the waypointsPanelData directly
```suggestion
  const waypointMenuData = useWaypointMenu(waypointsPanelData);
```

Reply:
Done